moncli/types.py

```diff
@@ -217,7 +180,79 @@ def _cast(self, value):
     def _export(self, value):
         return { 'rating': value}
 
-        
+
+class StatusType(MondayType):
+    native_type = str
+    allow_casts = (int,)
+    null_value = {}
+
+    def __init__(self, id: str = None, title: str = None, as_enum: type = None ,  *args, **kwargs):
+        if as_enum:
+            try:
+                enum_value = None
+                for value in list(as_enum):
+                    enum_value = value.name
+                    str(enum_value)
```

Comment:
Lines 193 to 195 are meant to verify that the value of the enum entries are str labels.  I would suggest getting rid of the try/except block entirely and go with the code below:

```python
for value in list(as_enum):
    if not isinstance(value.value, str)
        raise TypeError('Invalid value "{}" for status Enum "{}".'.format(value.value, value.__class__.__name__)
```